std_msgs/Header header
int32 n_hands
LeapHand[] hands